keyboards/phantagom/oled_mpad/keymaps/default/rules.mk

```diff
@@ -0,0 +1,24 @@
+# MCU name
```

Comment:
All of this should be removed, except what is needed (eg, what isn't present at the kb level).